src/status_im/protocol/web3/erc20.cljs

```diff
@@ -0,0 +1,48 @@
+(ns status-im.protocol.web3.erc20
```

Comment:
`protcol` ns is related only to messaging. Better to put this code to `s-i.utils.erc20`